apps/src/lib/node/ledger/protocol/transactions/ethereum_events/eth_msgs.rs

```diff
@@ -48,12 +49,8 @@ impl From<MultiSignedEthEvent> for EthMsgUpdate {
 pub struct EthMsg {
     /// The event being stored
     pub body: EthereumEvent,
-    /// The total voting power that's voted for this event across all epochs
-    pub voting_power: FractionalVotingPower,
-    /// The addresses of validators that voted for this event
-    pub seen_by: BTreeSet<Address>,
-    /// Whether this event has been acted on or not
-    pub seen: bool,
```

Comment:
A motivation for factoring these fields out now while at the same time implementing https://github.com/anoma/namada/issues/197 is that these three fields need to be read/updated but the body doesn't, and it should never change after an event is first recorded to storage. Most of the logic for updating vote tracking in storage can work with this `VoteTracking` struct now rather than an `EthMsg`.